content/intro-to-storybook/react/en/simple-component.md

```diff
@@ -111,7 +111,19 @@ When creating a story we use a base task (`taskData`) to build out the shape of
 
```

Comment:
I'm going to merge this, and I'll do that separately.

Reply:
@jonniebigodes I don't think we want to change our chapter naming from `simple` to `core`. "Simple" is the right word because it describes a low-complexity component. I don't think "core" communicates that as well.

Reply:
@domyen probably the word choice is not the best, but this change was taken from the "Gatsby documentation's" writing style which has been advocated by their learning team. 
I thought it could be a good addition to the tutorial aswell. 
The reasoning behind this is, if the tutorial is meant to introduce/educate people on how to use Storybook and with that drive the adoption as much as possible i think that lowering the entry level by avoiding as much as possible the usage of words like "basic", "simple" and some other ones. Then the documentation might benefit from it. I'll defer to both you and @tmeasday on how to proceed on this.

Reply:
No worries! In this case, "simple" describes the property of the component in that it's low complexity. We're using simple like "atomic" (we don't use atomic because it's too specific though 😅). 

I think Gatsby's style guide avoids using "simple", "basic", and "just" when describing things like workflows or instructions because not everything is simple to all skill levels. I generally agree with it, but I don't think that's the case here. 👍

Thanks for your attention to detail!